src/index.js

```diff
@@ -0,0 +1,52 @@
+import _ from 'lodash';
+import './style.css';
+
+function component() {
+  const element = document.createElement('div');
+
+  // Lodash, currently included via a script, is required for this line to work
+  element.innerHTML = _.join(['', ''], ' ');
+
+  return element;
+}
+
+document.body.appendChild(component());
+
+const data = [
+  {
+    description: 'make practice of coding',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'go to the swimming',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'online meeting',
+    completed: true,
+    index: 1,
+  },
+];
+const todoWrapper = document.querySelector('.new-task');
+// const newToDo = document.querySelector('.input-text');
+// const addToDo = document.querySelector('.btn-add');
+// const info = [];
```

Comment:
- [ Optional ] Kindly remove the commented codes from your code-base ,the commented code adds confusion with no real benefit and makes your code-base unclean .